src/views/tutorial-view/utils/variants/index.ts

```diff
@@ -25,6 +26,16 @@ export function getVariantParam(
 	return `${slug}:${optionSlug}`
 }
 
+export function handleVariantCookie(slug: string, optionSlug: string) {
+	const key = `variant-${slug}`
```

Comment:
❓ can we store all of the variant data in a single cookie? It'll give us more control in the event we end up changing a variant slug, and keeps the number of cookies we're setting in check. We'd store a stringified object like: `{ [slug]: optionSlug }`

Reply:
I could also use the uuid as the key to work around the slug issue, but good point on the single cookie, will look into that.

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/1953/commits/eef1fe7b7acce97dc8eb8312b6bc8fbae30dbafb! Still on the fence re: using the uuid, worried that might make debugging more difficult, but is a better long term identifier.

Reply:
Yeah, fair. Starting with a slug seems reasonable for now

Reply:
[Looping in this slack convo for shared context,](https://hashicorp.slack.com/archives/C050X5P9WFJ/p1684876960651469) Brandon agrees that slug is the right way to go.

<img width="1185" alt="Screenshot 2023-05-23 at 2 32 08 PM" src="https://github.com/hashicorp/dev-portal/assets/36613477/1c91e394-ef5c-42e0-9af6-ab560ac5e5ef">